code/game/objects/items/weapons/tools/wrenches.dm

```diff
@@ -13,17 +13,19 @@
 
 /obj/item/weapon/tool/wrench/improvised
 	name = "sheet spanner"
-	desc = "A flat bit of metal with some usefully shaped holes cut into it."
+	desc = "A flat bit of metal with some usefully shaped holes cut into it. Would perform better than regular wrench with some tool mods investment."
```

Comment:
"A flat bit of metal with some usefully shaped holes cut into it. Would perform better than a regular wrench with some tool mods investment."